services/brig/deb/opt/brig/templates/en/user/email/verification-login.txt

```diff
@@ -0,0 +1,16 @@
+[${brand_logo}]
```

Comment:
A corresponding html email should also be included; and ideally through adding the email contents to https://github.com/wireapp/wire-emails first, then re-importing through fetch.py (that would include doing a little bit of maintenance work also)